include/igl/MshLoader.cpp

```diff
@@ -16,6 +16,12 @@
 
 #include <string.h>
 
+#ifdef OSS_FUZZ
+#define NO_ABORT_ON_THROW 1;
```

Comment:
I do not get why you need to define a special behavior for the fuzz test. Most libigl code is written in C++, and throws exceptions upon error. We cannot go around and start replacing all our throw with a simple return.

Reply:
This is a suggested solution to the following problem: runtime errors, `abort`, `exit` are fuzz blockers. 
Another solution could be to use a custom libigl_exit(exit_type) and then consider if we are fuzzing inside it. This is also merely a suggestion.
Nonetheless, I do have some bandwidth for this if it could help improve the fuzzability of Libigl.

Reply:
Can't you simply wrap the libigl function call with a try/catch in the `igl_fuzzer.cpp` test file? If exceptions are a blocker for fuzzing, you can't even call STL functions anymore, since they can raise standard exceptions. I would like to find a solution that is less intrusive to the libigl codebase.